test/system/sessions_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class SessionsTest < ApplicationSystemTestCase
   setup do
-    create :user
+    @superadmin = create :user_with_profile
```

Comment:
This is a session test without profile stuff, so we don't depend on the user to be a superadmin, just use `@user = create :user`